### PR TITLE
Consolidate usage of tier names in the repo

### PR DESCRIPTION
**Pull request by kuszz**
solve the checkstyle errors #586

### CODE REVIEW

File: core/server/master/src/test/java/alluxio/master/file/FileSystemMasterTest.java [hunk 1]
```diff
@@ -2706,15 +2710,21 @@ private void startServices() throws Exception {
...
+        ImmutableMap.of(Constants.MEDIUM_MEM, (long) Constants.KB,
+                Constants.MEDIUM_SSD, (long) Constants.KB), ImmutableMap.of(),
         new HashMap<String, StorageList>(), RegisterWorkerPOptions.getDefaultInstance());
     mWorkerId2 = mBlockMaster.getWorkerId(
         new WorkerNetAddress().setHost("remote").setRpcPort(80).setDataPort(81).setWebPort(82));
-    mBlockMaster.workerRegister(mWorkerId2, Arrays.asList("MEM", "SSD"),
-        ImmutableMap.of("MEM", (long) Constants.MB, "SSD", (long) Constants.MB),
-        ImmutableMap.of("MEM", (long) Constants.KB, "SSD", (long) Constants.KB), ImmutableMap.of(),
+    mBlockMaster.workerRegister(mWorkerId2,
+            Arrays.asList(Constants.MEDIUM_MEM, Constants.MEDIUM_SSD),
```
Comment (maobaolong) [inline core/server/master/src/test/java/alluxio/master/file/FileSystemMasterTest.java:2723]: Fix the wrap indent  from 8 to 4 to keep consistent with others

File: core/common/src/main/java/alluxio/conf/PropertyKey.java [hunk 2]
```diff
@@ -72,6 +72,7 @@
    * values among the nodes. For example, it would show error if the Alluxio property for aws secret
    * access key has some value on a master node and a different value on one of the worker nodes.
    */
+
```
Comment (maobaolong) [inline core/common/src/main/java/alluxio/conf/PropertyKey.java:75]: Remove this empty line.

File: core/common/src/test/java/alluxio/conf/InstancedConfigurationTest.java [hunk 3]
```diff
@@ -995,7 +995,8 @@ public void testDeprecatedKeysNotLogged() {
 
   @Test
   public void unknownTieredStorageAlias() throws Exception {
-    for (String alias : Arrays.asList("mem", "ssd", "hdd", "unknown")) {
+    for (String alias : Arrays.asList(Constants.MEDIUM_MEM,
+            Constants.MEDIUM_SSD, "hdd", "unknown")) {
```
Comment (maobaolong) [inline core/common/src/test/java/alluxio/conf/InstancedConfigurationTest.java:999]: Fix the wrap indent from 8 to 4 to keep consistent with others

File: core/server/common/src/test/java/alluxio/AbstractStorageTierAssocTest.java [hunk 4]
```diff
@@ -74,7 +74,8 @@ public void masterWorkerConfConstructor() throws Exception {
    */
   @Test
   public void storageAliasListConstructor() {
-    List<String> orderedAliases = Arrays.asList("MEM", "HDD", "SOMETHINGELSE", "SSD");
+    List<String> orderedAliases = Arrays.asList(Constants.MEDIUM_MEM,
+            Constants.MEDIUM_HDD, "SOMETHINGELSE", Constants.MEDIUM_SSD);
```
Comment (maobaolong) [inline core/server/common/src/test/java/alluxio/AbstractStorageTierAssocTest.java:78]: Fix the wrap indent from 8 to 4 to keep consistent with others

File: core/server/common/src/test/java/alluxio/AbstractStorageTierAssocTest.java [hunk 5]
```diff
@@ -95,14 +96,18 @@ public void storageAliasListConstructor() {
     // Validate intersections.
     List<Pair<BlockStoreLocation, BlockStoreLocation>> intersections =
         workerAssoc.intersectionList();
-    Assert.assertEquals(intersections.get(0).getFirst(), BlockStoreLocation.anyDirInTier("MEM"));
-    Assert.assertEquals(intersections.get(0).getSecond(), BlockStoreLocation.anyDirInTier("HDD"));
-    Assert.assertEquals(intersections.get(1).getFirst(), BlockStoreLocation.anyDirInTier("HDD"));
+    Assert.assertEquals(intersections.get(0).getFirst(),
+            BlockStoreLocation.anyDirInTier(Constants.MEDIUM_MEM));
```
Comment (maobaolong) [inline core/server/common/src/test/java/alluxio/AbstractStorageTierAssocTest.java:100]: Fix the wrap indent from 8 to 4 to keep consistent with others

File: core/server/master/src/test/java/alluxio/master/block/BlockMasterTest.java [hunk 6]
```diff
@@ -136,11 +136,15 @@ public void countBytes() throws Exception {
...
     long worker1 = mBlockMaster.getWorkerId(NET_ADDRESS_1);
     long worker2 = mBlockMaster.getWorkerId(NET_ADDRESS_2);
-    List<String> tiers = Arrays.asList("MEM", "SSD");
-    Map<String, Long> worker1TotalBytesOnTiers = ImmutableMap.of("MEM", 10L, "SSD", 20L);
-    Map<String, Long> worker2TotalBytesOnTiers = ImmutableMap.of("MEM", 1000L, "SSD", 2000L);
-    Map<String, Long> worker1UsedBytesOnTiers = ImmutableMap.of("MEM", 1L, "SSD", 2L);
-    Map<String, Long> worker2UsedBytesOnTiers = ImmutableMap.of("MEM", 100L, "SSD", 200L);
+    List<String> tiers = Arrays.asList(Constants.MEDIUM_MEM, Constants.MEDIUM_SSD);
+    Map<String, Long> worker1TotalBytesOnTiers =
+            ImmutableMap.of(Constants.MEDIUM_MEM, 10L, Constants.MEDIUM_SSD, 20L);
```
Comment (maobaolong) [inline core/server/master/src/test/java/alluxio/master/block/BlockMasterTest.java:141]: Fix the wrap indent from 8 to 4 to keep consistent with others

File: core/server/master/src/test/java/alluxio/master/block/BlockMasterTest.java [hunk 7]
```diff
@@ -216,16 +220,18 @@ public void removeBlockTellsWorkersToRemoveTheBlock() throws Exception {
     // Create a worker with a block.
     long worker1 = mBlockMaster.getWorkerId(NET_ADDRESS_1);
     long blockId = 1L;
-    mBlockMaster.workerRegister(worker1, Arrays.asList("MEM"), ImmutableMap.of("MEM", 100L),
-        ImmutableMap.of("MEM", 0L), NO_BLOCKS_ON_LOCATION, NO_LOST_STORAGE,
+    mBlockMaster.workerRegister(worker1, Arrays.asList(Constants.MEDIUM_MEM),
+            ImmutableMap.of(Constants.MEDIUM_MEM, 100L),
```
Comment (maobaolong) [inline core/server/master/src/test/java/alluxio/master/block/BlockMasterTest.java:224]: Fix the wrap indent from 8 to 4 to keep consistent with others

File: core/server/master/src/test/java/alluxio/master/block/BlockMasterTest.java [hunk 8]
```diff
@@ -216,16 +220,18 @@ public void removeBlockTellsWorkersToRemoveTheBlock() throws Exception {
...
+        ImmutableMap.of(Constants.MEDIUM_MEM, 0L), NO_BLOCKS_ON_LOCATION, NO_LOST_STORAGE,
         RegisterWorkerPOptions.getDefaultInstance());
-    mBlockMaster.commitBlock(worker1, 50L, "MEM", "MEM", blockId, 20L);
+    mBlockMaster.commitBlock(worker1, 50L,
+            Constants.MEDIUM_MEM, Constants.MEDIUM_MEM, blockId, 20L);
```
Comment (maobaolong) [inline core/server/master/src/test/java/alluxio/master/block/BlockMasterTest.java:228]: Fix the wrap indent from 8 to 4 to keep consistent with others

File: core/server/master/src/test/java/alluxio/master/block/BlockMasterTest.java [hunk 9]
```diff
@@ -236,28 +242,30 @@ public void registerCleansUpOrphanedBlocks() throws Exception {
     // Create a worker with unknown blocks.
     long worker = mBlockMaster.getWorkerId(NET_ADDRESS_1);
     List<Long> orphanedBlocks = Arrays.asList(1L, 2L);
-    Map<String, Long> memUsage = ImmutableMap.of("MEM", 10L);
-    mBlockMaster.workerRegister(worker, Arrays.asList("MEM"), ImmutableMap.of("MEM", 100L),
+    Map<String, Long> memUsage = ImmutableMap.of(Constants.MEDIUM_MEM, 10L);
+    mBlockMaster.workerRegister(worker, Arrays.asList(Constants.MEDIUM_MEM),
+            ImmutableMap.of(Constants.MEDIUM_MEM, 100L),
```
Comment (maobaolong) [inline core/server/master/src/test/java/alluxio/master/block/BlockMasterTest.java:247]: Fix the wrap indent from 8 to 4 to keep consistent with others

File: core/server/master/src/test/java/alluxio/master/block/BlockMasterTest.java [hunk 10]
```diff
@@ -236,28 +242,30 @@ public void registerCleansUpOrphanedBlocks() throws Exception {
...
         memUsage, ImmutableMap.of(BLOCK_LOCATION, orphanedBlocks), NO_LOST_STORAGE,
         RegisterWorkerPOptions.getDefaultInstance());
 
     // Check that the worker heartbeat tells the worker to remove the blocks.
-    alluxio.grpc.Command heartBeat = mBlockMaster.workerHeartbeat(worker, null, memUsage, NO_BLOCKS,
-        NO_BLOCKS_ON_LOCATION, NO_LOST_STORAGE, mMetrics);
+    alluxio.grpc.Command heartBeat = mBlockMaster.workerHeartbeat(worker, null,
+            memUsage, NO_BLOCKS, NO_BLOCKS_ON_LOCATION, NO_LOST_STORAGE, mMetrics);
```
Comment (maobaolong) [inline core/server/master/src/test/java/alluxio/master/block/BlockMasterTest.java:253]: Fix the wrap indent from 8 to 4 to keep consistent with others

File: core/server/master/src/test/java/alluxio/master/block/BlockMasterTest.java [hunk 11]
```diff
@@ -236,28 +242,30 @@ public void registerCleansUpOrphanedBlocks() throws Exception {
...
 
   @Test
   public void workerHeartbeatUpdatesMemoryCount() throws Exception {
     // Create a worker.
     long worker = mBlockMaster.getWorkerId(NET_ADDRESS_1);
-    Map<String, Long> initialUsedBytesOnTiers = ImmutableMap.of("MEM", 50L);
-    mBlockMaster.workerRegister(worker, Arrays.asList("MEM"), ImmutableMap.of("MEM", 100L),
+    Map<String, Long> initialUsedBytesOnTiers = ImmutableMap.of(Constants.MEDIUM_MEM, 50L);
+    mBlockMaster.workerRegister(worker, Arrays.asList(Constants.MEDIUM_MEM),
+            ImmutableMap.of(Constants.MEDIUM_MEM, 100L),
```
Comment (maobaolong) [inline core/server/master/src/test/java/alluxio/master/block/BlockMasterTest.java:263]: Fix the wrap indent from 8 to 4 to keep consistent with others

File: core/server/master/src/test/java/alluxio/master/block/BlockMasterTest.java [hunk 12]
```diff
@@ -269,14 +277,17 @@ public void workerHeartbeatUpdatesMemoryCount() throws Exception {
   public void workerHeartbeatUpdatesRemovedBlocks() throws Exception {
     // Create a worker.
     long worker = mBlockMaster.getWorkerId(NET_ADDRESS_1);
-    mBlockMaster.workerRegister(worker, Arrays.asList("MEM"), ImmutableMap.of("MEM", 100L),
-        ImmutableMap.of("MEM", 0L), NO_BLOCKS_ON_LOCATION, NO_LOST_STORAGE,
+    mBlockMaster.workerRegister(worker, Arrays.asList(Constants.MEDIUM_MEM),
+            ImmutableMap.of(Constants.MEDIUM_MEM, 100L),
```
Comment (maobaolong) [inline core/server/master/src/test/java/alluxio/master/block/BlockMasterTest.java:281]: Fix the wrap indent from 8 to 4 to keep consistent with others

File: core/server/master/src/test/java/alluxio/master/meta/AlluxioMasterRestServiceHandlerTest.java [hunk 13]
```diff
@@ -90,14 +91,18 @@
...
-  private static final Map<String, Long> WORKER1_TOTAL_BYTES_ON_TIERS = ImmutableMap.of("MEM", 10L,
-      "SSD", 20L);
-  private static final Map<String, Long> WORKER2_TOTAL_BYTES_ON_TIERS = ImmutableMap.of("MEM",
-      1000L, "SSD", 2000L);
-  private static final Map<String, Long> WORKER1_USED_BYTES_ON_TIERS = ImmutableMap.of("MEM", 1L,
-      "SSD", 2L);
-  private static final Map<String, Long> WORKER2_USED_BYTES_ON_TIERS = ImmutableMap.of("MEM", 100L,
-      "SSD", 200L);
+  private static final Map<String, Long> WORKER1_TOTAL_BYTES_ON_TIERS =
+          ImmutableMap.of(Constants.MEDIUM_MEM, 10L,
```
Comment (maobaolong) [inline core/server/master/src/test/java/alluxio/master/meta/AlluxioMasterRestServiceHandlerTest.java:95]: Fix the wrap indent from 8 to 4 to keep consistent with others

**Review by maobaolong — CHANGES_REQUESTED** [review]
There are so many wrap line indent issue,  i just left comments within some of them, please take a complete look at this PR, ensure you fix all the wrap line issue.

**Comment by kuszz** [comment]
@LuQQiu @maobaolong  Thanks for pointing out the mistakes，please approve the pr , I have build and run in my machine,  thanks

**Comment by LuQQiu** [comment]
@kuszz Every time you make changes to the PR, the PR build will be retriggered. Because you are the first-time contributor, the build needs to be approved to be run. Need to PR build to be passed to merge the PR

**Comment by LuQQiu** [comment]
A good convention is that try not to use force-push unless necessary. Through normal push, reviewers can see your changes according to the reviews instead of need to review the whole PR again. No worry for this PR but please follow in the next PRs

**Comment by LuQQiu** [comment]
alluxio-bot, merge this please

**Comment by LuQQiu** [comment]
Congrats! @kuszz This PR is actually much more complicated than a new contributor PR, you have done a great job! Thanks a lot for making those useful changes!

**Comment by kuszz** [comment]
> A good convention is that try not to use force-push unless necessary. Through normal push, reviewers can see your changes according to the reviews instead of need to review the whole PR again. No worry for this PR but please follow in the next PRs

Thanks for your help and advice, I will be careful to follow the convention in the next PRs

**Pull request merged**